src/routes/RemediationDetails.js

```diff
@@ -74,10 +90,15 @@ class RemediationDetails extends Component {
 
         const { entitlements } = await window.insights.chrome.auth.getUser();
 
+        this.loadPlaybookRuns();
+
         this.setState({
             isEntitled: entitlements.smart_management.is_entitled
         });
 
+        fetch(`${window.location.origin}/api/sources/v2.0/endpoints?filter[receptor_node][not_nil]`)
```

Comment:
Can we put this to App.js so that we do not flood sources API with request on each page load (similarly to RBAC)? Also, might be better to put the actual fetch function call together with the URL definition to api.js and import it from there.